modules/virt-preventing-migration-during-eus-update.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+:_content-type: PROCEDURE
+[id="virt-preventing-migration-during-eus-update_{context}"]
+= Preventing VMs from migrating during an EUS-to-EUS update
+
+To reduce downtime when updating to the next Extended Update Support (EUS) version, prevent virtual machines (VMs) from migrating when their hosting `virt-launcher` pod is updated.
+
+.Prerequisites
+
+* You are running an EUS version of {product-title} and want to update to the next EUS version. You have not yet updated to the odd-numbered version in between.
+
+* You read "Preparing to perform an EUS-to-EUS update" and learned the caveats and requirements that pertain to your {product-title} cluster.
+
+* You paused the worker nodes' machine config pools as directed by the {product-title} documentation.
+
+.Procedure
+
+. Back up the current `workloadUpdateMethods` configuration by running the following command:
++
+[source,terminal]
+----
+$ WORKLOAD_UPDATE_METHODS=$(oc get kv kubevirt-kubevirt-hyperconverged -n openshift-cnv -o jsonpath='{.spec.workloadUpdateStrategy.workloadUpdateMethods}')
+----
+
+. Prevent the workloads from being disturbed after the update by removing all workload update methods:
```

Comment:
`after the update` you might want to elaborate here. We are turning off workload update methods, to avoid updating workloads prior to reaching target CNV EUS version